### PR TITLE
Payubiz Success/Cancel stage handled

### PR DESCRIPTION
**Pull request by sakinaboriwala**
- If user Commpletes the payment Successfully then they are redirected
to the Application to the Order Placed Successfully Screen.

- If user cancels the Payment then they are redirected to the Order
Failed Screen

### CODE REVIEW

File: lib/scoped-models/cart.dart [hunk 1]
```diff
@@ -199,7 +197,10 @@ mixin CartModel on Model {
     Map<String, String> headers = await getHeaders();
 
     Map<String, dynamic> paymentPayload = {
-      'payment': {'payment_method_id': 3, 'amount': order.itemTotal}
+      'payment': {
+        'payment_method_id': paymentMethodId,
```
Comment (gopalshimpi) [inline lib/scoped-models/cart.dart:201]: @sakina1403  is the payment method ID coming from back-end ?

File: lib/utils/constants.dart [hunk 2]
```diff
@@ -1,5 +1,7 @@
 class Settings {
   static const String SERVER_URL = 'https://ofypets.indiepet.co.in/';
+  static const String WEB_URL = 'https://beta.ofypets.com/';
+  static const String CANCEL_URL = 'https://test.payu.in/cancel';
```
Comment (gopalshimpi) [inline lib/utils/constants.dart]: Please add url  for production payubiz cancel action.

**Review by gopalshimpi — CHANGES_REQUESTED** [review]

File: lib/scoped-models/cart.dart (hunk 1 above)
Comment (sakinaboriwala) [reply lib/scoped-models/cart.dart:201]: @gopalshimpi Not yet. I have the API for it, will integrate it tomorrow.

File: lib/utils/constants.dart (hunk 2 above)
Comment (sakinaboriwala) [reply lib/utils/constants.dart]: Okay

File: lib/scoped-models/cart.dart (hunk 1 above)
Comment (gopalshimpi) [reply lib/scoped-models/cart.dart:201]: Keep It in Mind Before deploying to Production.

**Comment by gopalshimpi** [comment]
@sakina1403 Can you please use our standard template for PR's as its open source repository. Same for @Abdulrehman53 & @khuntdhams, Thanks!

**Review by gopalshimpi — APPROVED** [review]
LGTM: +1

**Pull request merged**